pandas/core/arrays/datetimes.py

```diff
@@ -819,7 +819,7 @@ def tz_convert(self, tz):
         dtype = tz_to_dtype(tz)
         return self._simple_new(self.asi8, dtype=dtype, freq=self.freq)
 
-    def tz_localize(self, tz, ambiguous="raise", nonexistent="raise"):
+    def tz_localize(self, tz, ambiguous="raise", nonexistent="raise"):  # noqa
```

Comment:
is this related to the rest of the PR?

Reply:
that shouldn't have been left in, the pydocstyle cli is crashing on that docstring. will need to raise an issue(look into this further), if we want to use this tool for cli.

will remove.